src/maintenance/repositories/restricted.md

```diff
@@ -0,0 +1,37 @@
+# Restricted Packages
+
+Void offers some packages that are officially maintained, but not distributed.
+These packages are marked as restricted and must be built from their
+[void-packages](https://github.com/void-linux/void-packages) template locally.
+
+Packages can be restricted from distribution by either the upstream author or
+Void. Void reserves the right to restrict distribution of any package for
+effectively any reason, massive size being the most common. Another common
+reason is restrictive licensing that does not allow third-party redistribution
+of source or binary packages.
+
+## Building manually
```

Comment:
Oh, I thought I got through all the notes last time, oops. I'm referring people to the void-packages documentation, but still mentioning the `XBPS_ALLOW_RESTRICTED` flag, since it's not very obvious at all in the documentation, but removed the command line to do it .